coderd/notifications/manager_test.go

```diff
@@ -182,6 +182,25 @@ func TestStopBeforeRun(t *testing.T) {
 	}, testutil.WaitShort, testutil.IntervalFast)
 }
 
+func TestRunStopRace(t *testing.T) {
+	t.Parallel()
+
+	// SETUP
+
+	// nolint:gocritic // Unit test.
+	ctx := dbauthz.AsSystemRestricted(testutil.Context(t, testutil.WaitSuperLong))
```

Comment:
nit: `testutil.WaitSuperLong` - can we shorten the period?

Reply:
For sure, this was copy 🍝